src/templates/vue/src/components/NetworkStatus.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div>
+    <div v-if="!online" class="msg offline-msg">
+        <div class="icon">
+          <span></span>
+        </div>
+        <div class="content">          
+          <div><strong>Offline : </strong>Connection lost! You are not connected to internet</div>
+        </div>
+    </div>
+    <div v-if="isBackOnline" class="msg online-msg">
+        <div class="icon">
+          <img src="https://img.icons8.com/office/40/000000/high-connection.png"/>
+        </div>
+        <div class="content">
+          <div><strong>Online : </strong>Back to online</div>
+        </div>
+    </div>
+  </div>
+</template>
+
+<script>
+const ONLINE = "online"
+const OFFLINE = "offline"
+
+export default {
+  name : "NetworkStatus",
+  data() {
+    return {
+      online: navigator.onLine,
```

Comment:
onLine is a single word so should be `online`

Reply:
By default, we are getting onLine prop from navigator